lib/ex_force.ex

```diff
@@ -175,7 +175,7 @@ defmodule ExForce do
           sobject_name,
           field_name,
           list(field_name)
-        ) :: {:ok, SObject.t()} | {:error, any}
```

Comment:
You can write it like this:

```elixir
{:ok, SObject.t() | QueryResult.t()} | {:error, any}
```

I prefer the above, as it reads as "there are ok or error cases, and ok has two possible types" - instead of "there are three cases - ok with SObject, ok with QueryReesult, and error".

Reply:
Done!